apps/public/src/app/(general)/bijlage/Attachments.test.tsx

```diff
@@ -0,0 +1,138 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { http, HttpResponse } from 'msw'
+
+import { ENDPOINTS } from 'apps/public/src/mocks/endpoints'
+import { server } from 'apps/public/src/mocks/node'
+
+import { Attachments } from './Attachments'
+
+vi.mock('./actions', () => ({
+  redirectToNextPage: vi.fn(),
+}))
+
+const defaultProps = {
+  meldingId: 1,
+  token: 'mock-token',
+}
+
+describe('Attachments', () => {
+  beforeAll(() => {
+    global.URL.createObjectURL = vi.fn(() => 'mocked-url')
+  })
+
+  it('should render correctly', () => {
+    render(<Attachments {...defaultProps} />)
+
+    const backLink = screen.getByRole('link', { name: 'Vorige vraag' })
+    const header = screen.getByRole('heading', { name: 'Foto’s' })
+    const fileUpload = screen.getByLabelText(/Selecteer bestanden/i)
+
+    expect(backLink).toBeInTheDocument()
+    expect(header).toBeInTheDocument()
+    expect(fileUpload).toBeInTheDocument()
+  })
+
+  it('should show file names when a file is uploaded', async () => {
+    const user = userEvent.setup()
+
+    render(<Attachments {...defaultProps} />)
+
+    const fileInput = screen.getByLabelText(/Selecteer bestanden/i) as HTMLInputElement
+
+    const file = new File(['dummy content'], 'Screenshot 2025-02-10 at 08.29.41.png', { type: 'image/png' })
+
+    await user.upload(fileInput, [file])
+
+    const fileName1 = screen.getByText('Screenshot 2025-02-10 at 08.29.41.png')
+
+    expect(fileName1).toBeInTheDocument()
+  })
+
+  it('should delete a file with the delete button', async () => {
+    const user = userEvent.setup()
+
+    render(<Attachments {...defaultProps} />)
+
+    const fileInput = screen.getByLabelText(/Selecteer bestanden/i) as HTMLInputElement
+
+    const file = new File(['dummy content'], 'Screenshot 2025-02-10 at 08.29.41.png', { type: 'image/png' })
+
+    await user.upload(fileInput, [file])
+
+    const fileName1 = screen.getByText('Screenshot 2025-02-10 at 08.29.41.png')
+
+    expect(fileName1).toBeInTheDocument()
+
+    const deleteButtons = screen.getAllByRole('button')
+
+    await user.click(deleteButtons[0])
```

Comment:
This is a little confusing, there is only 1 delete button at this point, no?

```suggestion
    const deleteButton = screen.getByRole('button', { name: 'Verwijder' })

    await user.click(deleteButton)
```